### DISCUSSION TITLE
New beatmap card difficulty list doesn't show up when hovering

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#15907] [Q&A] [answered]
I don't know whether or not this is intentional, but when I hover over the bottom of a beatmap card ingame, there is no difficulty list like there is on the website. I know the changelog showed that this new version of osu!lazer had this feature, so I'm kind of confused.

**Comment by respektive** [comment, marked as answer]
https://github.com/ppy/osu/issues/14216

beatmap cards aren't fully done yet, the difficulty dropdown will come later on
